### PR TITLE
Translation of app.rst to spanish

### PR DESCRIPTION
**Pull request by ivanyaros**
I've added some notes to explain some ambiguous issues

### CODE REVIEW

File: es/core-libraries/app.rst [hunk 1]
```diff
@@ -1,15 +1,133 @@
...
-    `Github <https://github.com/cakephp/docs>`_ o utilizar el botón **Improve this Doc** para proponer directamente los cambios.
+.. php:class:: App
 
-    Usted puede hacer referencia a la versión en Inglés en el menú de selección superior
-    para obtener información sobre el tema de esta página.
+La clase App se encarga de la localización de recursos y de la 
+administración de rutas.
+
+Búsqueda de clases
+===============
```
Comment (markstory) [inline es/core-libraries/app.rst]: Title underlines need to be the same length as the text above them.

File: es/core-libraries/app.rst [hunk 2]
```diff
@@ -1,15 +1,133 @@
...
+
+.. php:staticmethod:: Plugin::path(string $plugin)
+
+Los plugins se localizan con el método Plugin. Por ejemplo, ``Plugin::path('DebugKit');``
+devuelve la ruta completa al plugin DebugKit::
+
+    $path = Plugin::path('DebugKit');
+
+Localización de temas (nota:'themes')
+=============================
```
Comment (markstory) [inline es/core-libraries/app.rst]: Title underlines need to be the same length as the text above them.

File: es/core-libraries/app.rst [hunk 3]
```diff
@@ -1,15 +1,133 @@
...
+
+Dado que los temas (nota:'themes') son también plugins, 
+se localizan con el método anterior, "Plugin".
+(nota:'Aquí se refiere a los themes que se pueden crear 
+para modificar el comportamiento del bake, generador de código.')
+
+Cargar archivos externos (nota: 'vendor')
+====================
```
Comment (markstory) [inline es/core-libraries/app.rst]: Title underlines need to be the same length as the text above them.

File: es/core-libraries/app.rst (hunk 1 above)
Comment (ivanyaros) [inline es/core-libraries/app.rst]: solved

**Comment by ivanyaros** [comment]
I am allowed to put some notes, my commentaries in the translation?
I've added some in several cases, to explain some issues.

**Comment by markstory** [comment]
Thanks :clap:

**Pull request merged**